lib/bridge.js

```diff
@@ -0,0 +1,165 @@
+			return response.body;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Gets a list of bridges.
+	 * @param {Object} params Query parameters for listing bridges
+	 * @param {Number} [params.page=0] Used for pagination to indicate the page requested for querying
+	 * a list of bridges.
+	 * If no value is specified the default is 0.
+	 * @param {Number} [params.size=25] Used for pagination to indicate the size of each page requested
+	 * for querying a list of bridges. If no value is specified the default value is 25 (maximum value 1000).
+	 * @param {Function} callback A callback with the list of bridges
+	 * @return {Promise} A promise for the list of bridges
+	 */
+	this.list = function (params, callback) {
+		return client.makeRequest({
+			path   : "bridges",
+			method : "GET",
+			qs     : params
+		})
+		.then(function (response) {
+			return response.body;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Update the bridge
+	 * @param  {String} bridgeId The ID of the bridge
+	 * @param {Object} params Changed parameters of the bridge
+	 * @param {Boolean} params.bridgeAudio Enable/Disable two way audio path (default = true).
+	 * @param {Array<String>} params.callIds The list of call ids in the bridge.
+	 * @param {Function} [callback] Callback with the newly created bridge
+	 * @return {BridgeResponse} A promise for the operation
+	 */
+	this.update = function (bridgeId, params, callback) {
+		return client.makeRequest({
+			path   : "bridges/" + bridgeId,
+			method : "POST",
+			body   : params
+		})
+		.then(function () {
+			return;
+		})
+		.asCallback(callback);
+	};
+
+	function audioApi (bridgeId, params, callback) {
+		return client.makeRequest({
+			path   : "bridges/" + bridgeId + "/audio",
+			method : "POST",
+			body   : params
+		})
+		.then(function (response) {
+			return response.body;
+		})
+		.asCallback(callback);
+	}
+
+	/**
+	 * Speak sentence to the bridge
+	 * @param {String} bridgeId The ID of the bridge
+	 * @param {String} sentence A sentence to speak to the bridge.
+	 * @param {Object} params Optional parameters for the operation.
+	 * @param {Function} [callback] Callback for the operation
+	 * @return {BridgeResponse} A promise for the operation
+	 */
+	this.speakSentence = function (bridgeId, sentence, params, callback) {
+		if (!params) {
+			params = {};
+		}
+		params.sentence = sentence;
+		return audioApi(bridgeId, params).asCallback(callback);
+	};
+
+	/**
+	 * Play audio url to the bridge
+	 * @param {String} bridgeId The ID of the bridge
+	 * @param {String} fileUrl Url to audio file to play.
+	 * @param {Object} params Optional parameters for the operation.
+	 * @param {Function} [callback] Callback for the operation
+	 * @return {BridgeResponse} A promise for the operation
+	 */
+	this.playAudio = function (bridgeId, fileUrl, params, callback) {
+		if (!params) {
+			params = {};
+		}
+		params.fileUrl = fileUrl;
+		return audioApi(bridgeId, params).asCallback(callback);
+	};
+
```

Comment:
Same as speaksentence... need to handle situation where callback is used with no parameters